src/content/docs/installation/installation_prepare.mdx

```diff
@@ -6,35 +6,29 @@ description: How to prepare CachyOS for installation
 import { Steps, Tabs, TabItem, Aside } from '@astrojs/starlight/components';
 import ImageComponent from '~/components/image-component.astro';
 
-<Aside type="tip" title="Getting ready for CachyOS.">
+<Aside type="tip" title="Getting Ready for CachyOS.">
    The following section will guide you through the steps to prepare your USB drive with CachyOS ISO image for installation and the necessary requirements.
 </Aside>
 
 ## System Requirements
 
-Before you begin with the preparation for installation, you need to make sure that your computer meets the following system requirements in order to run CachyOS.
+Before you begin with the preparation for installation, you need to make sure that your computer meets the following system requirements in order to run CachyOS. Bear in mind that our installation process uses online installation.
 
 <Tabs>
 
 <TabItem label="Minimum requirements">
 
 *   3 GB RAM
 *   30 GB of Storage Space (HDD/SSD)
-*   [x86-64-v1 capable CPU](/installation/installation_prepare#x86_64-microarchitecture-level-support) (64-bit)
-*   Integrated graphics card
-*   Stable internet connection
-
-:::note
-An unstable internet can lead to installation issues and long waits.
-:::
+*   Stable internet connection - unstable internet may cause prolonged period of installation or failed installs at worst
```

Comment:
Can you convert it back to a note or tip? i made that way to make even more obvious that more than a requirement